cardano-db-sync-extended/app/cardano-db-sync-extended.hs

```diff
@@ -37,9 +40,16 @@ main = do
         trce <- configureLogging params
         let MigrationDir migrationDir = enpMigrationDir params
         DB.runMigrations id True (DB.MigrationDir migrationDir) (Just $ DB.LogFileDir "/tmp")
-        runDbSyncNode trce extendedDbSyncNodePlugin params
 
--- -------------------------------------------------------------------------------------------------
+        -- Open up a connection and use it
+        pgconf <- DB.readPGPassFileEnv
+        let connectionString = DB.toConnectionString pgconf
+
+        DB.runIohkLogging trce $ withPostgresqlPool connectionString 10 $ \pool ->
+          withResource pool $ \backend ->
```

Comment:
I really do not think we need a pool here. In `db-sync` there is only ever a single thread operating on the database so a single connection should be sufficient (unless of course we have proof that using a pool provided better performance).